### DISCUSSION TITLE
Unable to get form context in external components inside form provider

### DISCUSSION THREAD

**Discussion by iaashish** [react-hook-form/react-hook-form#10099] [General]
I created a reusable component using react-hook-form. But when i include that component in a FormProvider which is another project,  reusable component is unable to retrieve the formcontext. Is this the existing issue?

**Comment by leapful** [comment]
@iaashish 

Please provide a codesandbox url to re-produce the issue. 

Without any code example or context how your other project setting for React Provider and Context, it's impossible to investigate the problem and no one knows if it's related to existing issue or not.
